administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -3,18 +3,30 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
+COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_SUCCESS="Successful registered to autoupdate service"
+COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_ERROR="Error while registering to autoupdate service: %s (%d)"
+COM_JOOMLAUPDATE_AUTOUPDATE_UNREGISTER_SUCCESS="Successful unregistered from autoupdate service"
+COM_JOOMLAUPDATE_AUTOUPDATE_UNREGISTER_ERROR="Error while unregistering from autoupdate service: %s (%d)"
 COM_JOOMLAUPDATE_CAPTIVE_HEADLINE="Confirm your credentials"
 COM_JOOMLAUPDATE_CHECKED_UPDATES="Checked for updates."
+COM_JOOMLAUPDATE_CONFIG_AUTO_UPDATE_DISABLED_LABEL="<span class=\"fa-fw me-2 fa fa-info-circle\"></span> Automated updates are only supported for the \"default\" channel and \"Minimum Stability\" \"Stable\"."
+COM_JOOMLAUPDATE_CONFIG_AUTOMATED_UPDATES_LABEL="Automated updates"
```

The UI language styleguide says that labels should be capitalised

```suggestion
COM_JOOMLAUPDATE_CONFIG_AUTOMATED_UPDATES_LABEL="Automated Updates"
```